packages/blocks/src/api/utils.js

```diff
@@ -161,19 +161,24 @@ export function getBlockLabel( blockType, attributes, context = 'visual' ) {
  * @param {Object}  attributes             The values of the block's attributes.
  * @param {?number} position               The position of the block in the block list.
  * @param {string}  [direction='vertical'] The direction of the block layout.
+ * @param {number} total The total number of blocks.
+ * @param {string} parentTitle In the event of a child block, this param gets the title of the parent block.
  *
  * @return {string} The block label.
  */
 export function getAccessibleBlockLabel(
 	blockType,
 	attributes,
 	position,
-	direction = 'vertical'
+	direction = 'vertical',
+	total,
+	parentTitle
```

Comment:
This function is used in a React Native file, which will need updating as well: https://github.com/WordPress/gutenberg/blob/da8555d258edbe676fa079fb51252f918ae032e1/packages/block-editor/src/components/block-list/block.native.js#L191-L195

The total here I understand corresponds to the block count, which comes from: https://github.com/WordPress/gutenberg/blob/da8555d258edbe676fa079fb51252f918ae032e1/packages/block-editor/src/store/selectors.js#L314